packages/icons/plugins/svg-dynamic-title-plugin.js

```diff
@@ -0,0 +1,45 @@
+// Plugin to add <title> and <desc> to SVG elements
+const elements = ['svg', 'Svg']
+
+const dynamicTitlePlugin = ({ types: t }) => {
+  const addTitle = {
+    JSXElement(path) {
+      if (
+        !elements.some(element =>
+          path.get('openingElement.name').isJSXIdentifier({ name: element })
+        )
+      ) {
+        return
+      }
+
+      //Remove the Default <title> and <desc> tags if exists
```

Comment:
Keep comment spacing consistent, such as L1.

```suggestion
      // Remove the Default <title> and <desc> tags if exists
```